.github/workflows/ocaml-ci.yml

```diff
@@ -40,5 +40,9 @@ jobs:
           opam exec -- make build
 
       - name: Run tests
-        run: opam exec -- make test
+        run: opam exec -- dune runtest --instrument-with bisect_ppx --force
 
+      - run: bisect-ppx-report send-to Coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PULL_REQUEST_NUMBER: ${{ github.event.number }}
```

Comment:
How do we see the coverage? Can we add a link to the coveralls page to the readme?

Reply:
I'll add a link, coverage can be seen at https://coveralls.io/github/xapi-project/nbd

Reply:
we can also remove the travis build status